proto/osmosis/gamm/pool-models/balancer/balancerPool.proto

```diff
@@ -102,6 +102,15 @@ message PoolAsset {
     (gogoproto.moretags) = "yaml:\"weight\"",
     (gogoproto.nullable) = false
   ];
+
+  //   Weight that is normalized s.t. the sum of all pool assets' weights
+  // equals 1. If the user provides this value, it is ignored. This should only
+  // ever be set by the state machine. This is left as a TODO for a future PR.
```

Comment:
Also, is there a safeguard against this being ever set by a user? Is that what the TODO is for?